schemas/Screen.json

```diff
@@ -7,21 +7,46 @@
     },
     "screenName": {
       "type": "string",
-      "enum": [
-        "Home",
-        "Room",
-        "RoomDirectory",
-        "User",
-        "Group",
-        "MyGroups",
-        "WebLoading",
-        "WebWelcome",
-        "WebLogin",
-        "WebRegister",
-        "WebForgotPassword",
-        "WebCompleteSecurity",
-        "WebE2ESetup",
-        "WebSoftLogout"
+      "oneOf": [
+        {"const": "Home", "description": "The Home tab on iOS | possibly the same on Android? | The Home space on Web?"},
```

Comment:
Thanks for the approval @novocaine 🙌
Just wanted to double check you're happy with this before I merge.